### ISSUE TITLE
Add a way for us to "check off" people who have won

### ISSUE THREAD

**Issue by jetpacktuxedo** [c-keys/raffle#3]
theoretically the system we used opens us up to people double-claiming prizes

**Comment by brandenbyers** [comment]
I suggest we create a new CSV with winners removed. That way we can run the script again on the second CSV if needed. And then on the third and so on.

**Comment by jetpacktuxedo** [comment]
Yeah, but that should be a part of the system. Either creating a csv with the original winners stripped out as a default function or maybe like a prompt to rerun or something.

**Comment by jameshly** [comment]
So re-raffling can be done pretty easily, but how are we going to mark off people who have claimed their prizes as we give prizes out?

**Comment by brandenbyers** [comment]
See above. I suggest creating new csv and html files. And call the function three times. So when the script is run, it automatically creates two extra CSV files and 3 total html files.

Original CSV file
Redraw CSV 1 (with original winners removed)
Redraw CSV 2 (with original and second redraw removed)

winners-1.html
winners-2.html
winners-3.html

That way we only need to run it once for all of our redraws. And we use the redraws I’d we need them.
